src/pages/project-list/count.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import { Badge, Heading5, CaptionBold } from 'styles/typography';
+import color from 'styles/colors';
+import { numberWithCommas } from 'utils';
+
+export enum GiveOrTake {
+  Give = 'give',
+  Take = 'take'
+}
+
+interface CountProps {
+  type: GiveOrTake;
+  amount: number;
+}
+
+const conditionalGap = css`
+  &:nth-child(2) {
+    margin-left: 17px;
+  }
+`;
+
+const PriceWrap = styled.div<{ isGive: boolean }>`
+  display: flex;
+  align-items: center;
+
+  color: ${({ isGive }) => (isGive ? color.red : color.green)};
+`;
+
+const Point = styled(Heading5)<{ isGive: boolean }>`
+  color: ${({ isGive }) => (isGive ? color.red : color.green)};
+`;
+
+const Icon = styled.div<{ isGive: boolean }>`
+  width: 12px;
+  height: 12px;
+  ${({ isGive }) =>
+    isGive
+      ? css`
+          background: url('./images/ico_give.png') center no-repeat;
+        `
+      : css`
+          background: url('./images/ico_take.png') center no-repeat;
+        `}
```

Comment:
`css` 함수 호출없이

```suggestion
background: url('./images/${({ isGive }) => (isGive ? 'ico_give' : 'ico_take')}.png') center no-repeat;
```

요렇게 해도 워킹하지 않나용???

Reply:
옷 네네! 훨씬 간단하네용 ㅎㅎ

Reply:
- ec61711